src/main/java/com/academy/terai/Controller/ApplicationController.java

```diff
@@ -0,0 +1,59 @@
+package com.academy.terai.Controller;
+
+import com.academy.terai.Model.Application;
+import com.academy.terai.Service.ApplicationService;
+import javassist.NotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/application")
+public class ApplicationController {
+
+    private final ApplicationService applicationService;
+
+    @Autowired
+    public ApplicationController(ApplicationService applicationService) {
+        this.applicationService = applicationService;
+    }
+
+
+    @GetMapping
+    public ResponseEntity<List<Application>> getAllApplications() {
+        List<Application> applications = applicationService.findAll();
```

Comment:
Risky practice to load all items from the database at once. Might be a bit challenging, but I'd suggest to use pagination. Just google 'Spring data pagination', you'd find many examples.
In short, it allows to load only expected amount of data (for one page) with each call.